### PR TITLE
ci: Benchmark against google-quiche

### PR DESCRIPTION
**Pull request by larseggert**

### CODE REVIEW

File: .github/workflows/bench.yml [hunk 1]
```diff
@@ -132,15 +156,21 @@ jobs:
               CMD=${CMD//_pacing/--no-pacing}
               EXT="$EXT-nopacing"
             fi
+            CMD=${CMD//_flags/$flags}
           }
 
           # See https://github.com/microsoft/msquic/issues/4618#issuecomment-2422611592
-          for mtu in 1504 65536; do
+          for mtu in 1504; do # We used to measure 65536 here, too, but that's somewhat unrealistic.
```
Comment (mxinden) [inline .github/workflows/bench.yml:163]: How about removing the loop, given that it iterates at most once. Would reduce the level of indentation.

**Review by mxinden — APPROVED** [review]
`neqo` to `neqo` with MTU of `1504` seems to have increased in this pull request. Is this noise or a regression?

Compare e.g. with https://github.com/mozilla/neqo/pull/2216 or https://github.com/mozilla/neqo/pull/2225.

Otherwise looks good to me. Ready to merge.

File: .github/workflows/bench.yml (hunk 1 above)
Comment (larseggert) [reply .github/workflows/bench.yml:163]: Can do. Was wondering if we wanted to put it back at some point.

**Comment by larseggert** [comment]
> `neqo` to `neqo` with MTU of `1504` client/server transfer seems to have increased in this pull request. Is this noise or a regression?

I don't know.

File: .github/workflows/bench.yml (hunk 1 above)
Comment (larseggert) [reply .github/workflows/bench.yml:163]: Should I remove the 65536 MTU everywhere? Would cut down the time on benches.

Comment (mxinden) [reply .github/workflows/bench.yml]: I assume we will focus on Internet paths for now, thus I suggest removing 64k MTU entirely.

**Comment by mxinden** [comment]
> [ja4 workflow](https://github.com/mozilla/neqo/pull/2218/commits/648a380933ac83a48ba45ecf0ad581dc6e8b6eb4)

Bad push @larseggert?

**Pull request merged**